lib/index.js

```diff
@@ -888,6 +885,301 @@ module.exports = () => {
         throw new Error('ec_seckey_tweak_mul', ret);
       }
 
+      /**
+       * @summary check if a point is valid on the curve
+       * @arg {Uint8Array} point - point to check
+       * @returns {boolean} true if valid, false otherwise
+       */
+      function isPoint(point) {
+        if (!point || !(point instanceof Uint8Array)) {
+          throw new TypeError('point must be a Uint8Array');
+        }
+
+        const pointPtr = charStar(point);
+        const res = Module.ccall(
+          'ec_is_point',
+          'number',
+          ['number', 'number'],
+          [pointPtr, point.length]
+        );
+        freeMalloc();
+        return res === 1;
+      }
+
+      /**
+       *  @summary compress a valid point
+       *  @arg {Uint8Array} point - point to compress
+       *  @returns {Uint8Array} compressed point
+       **/
+      // eslint-disable-next-line no-unused-vars
+      function pointCompress(point, _) {
+        if (!point || !(point instanceof Uint8Array)) {
+          throw new TypeError('point must be a Uint8Array');
+        }
+        const outputLen = 33;
+        const outputLenPtr = malloc(4);
+        Module.setValue(outputLenPtr, outputLen, 'i32');
```

Comment:
`size_t` is 8 bytes:
```suggestion
        const outputLenPtr = malloc(8);
        Module.setValue(outputLenPtr, outputLen, 'i64');
```